third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/user-prompting-checker.js

```diff
@@ -148,6 +150,11 @@ UserPromptingChecker.prototype.check = function(clientOrigin) {
       },
       this);
 
+  delete goog.global[`googleSmartCard_resolveModalDialog${
```

Comment:
This code should be moved to popup-opener.js, because this code would be the same for every piece of code that wants to show a popup